zivid_samples/scripts/sample_capture.py

```diff
@@ -16,23 +16,25 @@ def __init__(self):
 
         rospy.wait_for_service("/zivid_camera/capture", 30.0)
 
-        rospy.Subscriber("/zivid_camera/points", PointCloud2, self.on_points)
+        rospy.Subscriber("/zivid_camera/points/xyzrgba", PointCloud2, self.on_points)
```

Comment:
Could be slightly annoying to someone who already uses this topic name in their code that we change the topic name here. But I think it makes sense, since the pointcloud format has actually change to not contain contrast (and since there are also updates like changing the iris to aperature).
Anyway the needed changes are apparent in the samples + readme.

Reply:
Also we are adding one other "points" type "points/xyz" which does not have the color and is therefore more performant. So therefore I think it makes sense to rename the old one.

Reply:
I agree, resolving this one